src/sql/procedure/delete_raw_employee_data.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM raw_employee_data;
```

Comment:
This is actually not a procedure but just a SQL query statement and I think this also can be generalized. How about we add all such queries in a constant file and then use it? For example:

`DELETE_QUERY= "DELETE FROM {table_name}"`